CoreHistogramView/src/au/gov/asd/tac/constellation/views/histogram/rewrite/HistogramDisplay2.java

```diff
@@ -0,0 +1,792 @@
+/*
+ * Copyright 2010-2025 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.histogram.rewrite;
+
+import au.gov.asd.tac.constellation.utilities.clipboard.ConstellationClipboardOwner;
+import au.gov.asd.tac.constellation.utilities.font.FontUtilities;
+import au.gov.asd.tac.constellation.utilities.javafx.JavaFxUtilities;
+import au.gov.asd.tac.constellation.views.histogram.Bin;
+import au.gov.asd.tac.constellation.views.histogram.BinCollection;
+import au.gov.asd.tac.constellation.views.histogram.BinIconMode;
+import au.gov.asd.tac.constellation.views.histogram.BinSelectionMode;
+import java.awt.Color;
```

Comment:
Does this class still need to be used with an fx implementation? I note that use of this seems to eventually translate to converting to an fx color anyway so might as well use the fx color class right from the start